bukkit/src/main/java/us/myles/ViaVersion/bukkit/platform/BukkitViaAPI.java

```diff
@@ -107,7 +107,7 @@ public boolean isProtocolSupport() {
         return plugin.isProtocolSupport();
     }
 
-    public Map<UUID, UserConnection> getPortedPlayers() {
-        return Via.getManager().getPortedPlayers();
+    private Map<UUID, UserConnection> getPortedPlayers() {
+        return Via.getManager().getConnectedClients();
```

Comment:
Thoughts on just removing this method and replacing the calls with Via.getMananger().getConnectedClients()?

Reply:
Probably JVM would inline it and wouldn't be too relevant

Reply:
More or less still an unnecessary wrapper tho (with old naming)